deploy/helm/sumologic/values.yaml

```diff
@@ -242,6 +242,20 @@ sumologic:
     collector:
       otelcol:
         enabled: true
+      otelcloudwatch:
```

Comment:
I think we should mark it as experimental for two reasons:
- we are still evaluating the final shape
- we may need to introduce breaking changes in near future